tests/rest/source/app.d

```diff
@@ -460,6 +460,64 @@ class Example7 : Example7API {
 	}
 }
 
+/* --------- EXAMPLE 8 ---------- */
+
+@rootPathFromName
+interface Example8API
+{
+	// Methods need to be `@safe`:
+	@safe:
+
+	FooType constFoo (const FooType param);
+
+	FooType constRefFoo (const ref FooType param);
+
+	FooType inFoo (in FooType param);
+
+	FooType immutableFoo (immutable FooType param);
+	struct FooType {
+		int a;
+		string s;
```

Comment:
Now try with `int[]`

Reply:
Done but this will probably have to wait for the next compiler release to work.